_schemaV2.graphql

```diff
@@ -11712,6 +11717,11 @@ type PartnerArtworkGrid implements ArtworkContextGrid {
   title: String
 }
 
+type PartnerArtworksError {
```

Comment:
similar comment as `PartnerArtworks` naming, should this be `PartnerArtworksBulkEditErrors` or something?

Reply:
Updated to that, thanks!